charts/redpanda/templates/service.internal.yaml

```diff
@@ -15,33 +15,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */}}
 ---
-# This service is only used to create the DNS enteries for each pod in
-# the stateful set and allow the serviceMonitor to target the pods.
-# This service should not be used by any client application
```

Comment:
NIT
I feel like argument in this comment is not accurate, but could be moved to the `service_internal.go`.